### ISSUE TITLE
Identity Manager gets 403 when trying to query the API 

### ISSUE THREAD

**Issue by Vad1mo** [pomerium/pomerium#1372]
We updated from Pomerium 8.x to Helm 12.2.1/ pomerium v0.10.0  and now the users group are not fetched from azure AD or at least aren't visible. 

In accordance with the docs, I created the serviceAccount with serviceAccountYAML
```yaml
      - authenticate:
          idp:
            provider: azure
            url: {{ .Values.idp.url }}
            clientID: {{ .Values.idp.clientID }}
            clientSecret: {{ .Values.idp.clientSecret }}
            serviceAccountYAML:
              client_id: {{ .Values.idp.clientID }}
              client_secret: {{ .Values.idp.clientSecret }}
              directory_id: {{ .Values.idp.directoryID }}
```
The generated json is:

```json
idp_client_id: 686...
idp_client_secret: 033...
idp_service_account: base64 value of  {"client_id":"686....","client_secret":"033...","directory_id":"a9d..."}
```
For the reference the used directoryID is also contained in the Azure OpenID Connect url.

When now the users gos to https://authenticate.domain.com/.pomerium/ there are no groups listed for the user.

I made sure that the Azure app API   contains: **Directory.Read.All, Group.Read.All**

In the cache logs I see the following:

```sh
{"level":"warn","service":"identity_manager","error":"azure: error querying api: 403 Forbidden","time":"2020-09-04T12:44:41Z","message":"failed to refresh directory users and groups"}
```

```json
2020-09-04T14:34:41.411249728+02:00 {"level":"info","service":"databroker","type":"type.googleapis.com/user.User","id":"azure/f6c2c1c1-...","time":"2020-09-04T12:34:41Z","message":"get"}
2020-09-04T14:34:41.412017268+02:00 {"level":"info","service":"databroker","type":"type.googleapis.com/directory.User","id":"azure/f6c2c1c1....","time":"2020-09-04T12:34:41Z","message":"get"}
2020-09-04T14:44:41.323833363+02:00 {"level":"info","service":"identity_manager","time":"2020-09-04T12:44:41Z","message":"refreshing directory users"}
2020-09-04T14:44:41.505720666+02:00 {"level":"warn","service":"identity_manager","error":"azure: error querying api: 403 Forbidden","time":"2020-09-04T12:44:41Z","message":"failed to refresh directory users and groups"}
2020-09-04T14:44:41.505739982+02:00 {"level":"info","service":"identity_manager","user_id":"azure/f6c2c1c1","time":"2020-09-04T12:44:41Z","message":"refreshing user"}
2020-09-04T14:44:41.649441211+02:00 {"level":"info","service":"databroker","type":"type.googleapis.com/user.User","id":"azure/f6c2c1c1","time":"2020-09-04T12:44:41Z","message":"set"}
2020-09-04T14:45:11.39712096+02:00 {"level":"info","service":"envoy","name":"main","time":"2020-09-04T12:45:11Z","message":"shutting down parent after drain"}
```

**Comment by Vad1mo** [comment]
I just found the solution to the problem:

https://github.com/pomerium/pomerium/issues/1050#issuecomment-658721216

After adding `Group.Read.All Application` permission and redeployment  i see the groups fetched 

I takes a few min until its there after the restart.

**Comment by Vad1mo** [comment]
I think the docs need an update on that:

**Comment by desimone** [comment]
Hey @vad1mo,

I just checked the docs and it looks like the correct permissions are listed. Where you referencing something else that needs to change? 

> On the App registrations page, click API permissions. Click the Add a permission button and select Microsoft Graph API, select Delegated permissions. Under the Directory row, **select the checkbox for Group.Read.All.**

If I missed something, let me know!

Thanks,
Bobby

**Comment by Vad1mo** [comment]
Yes, it is missing:
This is how it currently is:
![image](https://user-images.githubusercontent.com/1492007/92334900-67778900-f092-11ea-8b5f-944d2c316f91.png)

This is how it should be:
![image](https://user-images.githubusercontent.com/1492007/92334948-bde4c780-f092-11ea-8bac-19694187f268.png)

New is:
**Group.Read.All  Application**

**Comment by desimone** [comment]
@Vad1mo Cheers. Do you mind if I use your screenshot in the new docs?

**Comment by Vad1mo** [comment]
here are better ones, 

you can choose, with or without dropshadow.


![Bildschirmfoto 2020-09-06 um 23 49 53](https://user-images.githubusercontent.com/1492007/92335895-b37afb80-f09b-11ea-88c8-ba612a53dd50.png)
![Bildschirmfoto 2020-09-06 um 23 47 12](https://user-images.githubusercontent.com/1492007/92335898-b675ec00-f09b-11ea-855f-f178c8c5dffe.png)

**Comment by desimone** [comment]
@Vad1mo thanks, this is really helpful. Working through this, I realized many of the azure screens have changed since I wrote this guide. I took this as an opportunity to clean up the guide, and remove unused permissions and settings.

One that jumps out at me is that I do not think `Group.Read.All` or `Directory.Read.All` are required for delegated permissions anymore, but rather just Applciation settings for `Group.Read.All` as you point out above. 

I'll cc  you shortly on my PR, if you'd be able to give it a review.
